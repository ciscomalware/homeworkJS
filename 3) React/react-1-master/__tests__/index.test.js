import render from '../src/app';

test('dom', () => {
  document.body.innerHTML = '<div id="container"></div>';
  render();
  expect(document.body.innerHTML).toMatchSnapshot();
});
