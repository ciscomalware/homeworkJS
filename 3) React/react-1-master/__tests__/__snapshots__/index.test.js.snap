// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dom 1`] = `"<div id=\\"container\\"><div class=\\"card-deck\\"><div class=\\"card\\"><div class=\\"card-header\\">Jon</div><div class=\\"card-body\\"><ul class=\\"list-group-flush\\"><li class=\\"list-item\\">20</li><li class=\\"list-item\\">jon@winter.com</li></ul></div></div><div class=\\"card\\"><div class=\\"card-header\\">Daenerys</div><div class=\\"card-body\\"><ul class=\\"list-group-flush\\"><li class=\\"list-item\\">19</li><li class=\\"list-item\\">daenerys@dragons.com</li></ul></div></div><div class=\\"card\\"><div class=\\"card-header\\">Eddard</div><div class=\\"card-body\\"><ul class=\\"list-group-flush\\"><li class=\\"list-item\\">45</li><li class=\\"list-item\\">eddard@winter.com</li></ul></div></div><div class=\\"card\\"><div class=\\"card-header\\">Cersei</div><div class=\\"card-body\\"><ul class=\\"list-group-flush\\"><li class=\\"list-item\\">35</li><li class=\\"list-item\\">cersei@shame.com</li></ul></div></div></div></div>"`;
